README.md

```diff
@@ -26,11 +26,12 @@ This is the Ruby project #3, from the Microverse's main curriculum and aims to b
 
 - Install Ruby
 
-- Run ruby bin/main.rb
+- Type 'gem install colorize' on terminal to enable colors
```

Comment:
- [ ] **OPTIONAL** For good practices (and to make your life easier in future projects), please try using a Gemfile to put all your dependencies there instead of asking users to install the gem. In this project, it is just 1 gem but imagine a more complex project. [Here you can see how to implement it](https://bundler.io/gemfile.html) in summary it is to create a file called `Gemfile` and inside put all the gems that your project uses, these gems will be then installed with a single `bundle install` instruction.